src/frontend/Context/AuthReducer.js

```diff
@@ -0,0 +1,37 @@
+export const AuthReducer = (state, action) => {
+  const { type, payload, error } = action;
+  switch (type) {
+    case "REQUEST_LOGIN":
+      return {
+        ...state,
+        loading: true,
+      };
+    case "LOGIN_SUCCESS":
+      return {
+        ...state,
+        loading: false,
+        message: "",
+        isUserLoggedIn: true,
+        encodedToken: action.encodedToken,
+        userDetails: action.userDetails,
+      };
+    case "LOGOUT":
+      localStorage.clear();
```

Comment:
you should only clear the local storage of this site. And not the other sites. 
So <code>localStorage.clear("tokenname)</code>

Reply:
Thanks for pointing out this. i can see changes in next commit

Reply:
done